README.md

```diff
@@ -69,3 +74,8 @@ This ssh config does a couple of interesting things -
 
 ## Telemetry
 There currently is some basic trace instrumentation using [honeycomb](https://www.honeycomb.io/). We use this internally to track usage, gather performance statistics, and error reporting. Telemetry is disabled without a honeycomb write key - which you must provide through the [config](pkg/config/config.go).
+
+## Common Errors
+
+### Unsafe RSA public key
```

Comment:
can the error message link to this?

Reply:
I kind of want to write better error handling logic/messaging and not just this readme

Reply:
probably in a separate pr